src/components/panel/panel.js

```diff
@@ -1125,11 +1158,16 @@ MdPanelRef.prototype._createPanel = function() {
           self._panelEl = angular.element(
               self._panelContainer[0].querySelector('.md-panel'));
 
-          // Add a custom CSS class.
+          // Add a custom CSS class to the panel element.
           if (self._config['panelClass']) {
             self._panelEl.addClass(self._config['panelClass']);
           }
 
+          // Handle click and touch events for the panel container.
+          if (self._config['propagateContainerEvents']) {
+            self._panelContainer.css('pointer-events', 'none');
```

Comment:
Not sure if this is applicable here, but just be aware that some of our code calls `$mdUtil.getParentWithPointerEvents()` and it will now skip this. I only mention because I ran into a similar issue earlier with the Speed Dial where it caused it to miss the intended parent.